commands/bootstrap/__tests__/__fixtures__/ci/packages/package-1/package-lock.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "package-1",
+  "version": "1.0.0",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+    "tiny-tarball": {
+      "version": "1.0.0",
+      "resolved": "http://artifactory.aws.athenahealth.com:80/api/npm/npm/tiny-tarball/-/tiny-tarball-1.0.0.tgz",
```

Comment:
This should be resolved to the public registry, `https://registry.npmjs.org/`

Reply:
Oops :).  Nice catch, fixed...